rasa_core/broker.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import logging
+
+import pika
+
+logger = logging.getLogger(__name__)
+
+
+class EventBroker(object):
+    def process(self, body):
```

Comment:
Should get a short description (and I'd name the parameter `event` as you did in the producer implementation).